SafeguardDevOpsService/Logic/SafeguardLogic.cs

```diff
@@ -738,9 +766,9 @@ public string GetCSR(int? size, string subjectName, CertificateType certificateT
                 }
 
                 var builder = new StringBuilder();
-                builder.AppendLine("-----BEGIN CERTIFICATE REQUEST-----\r\n");
+                builder.AppendLine("-----BEGIN CERTIFICATE REQUEST-----");
```

Comment:
Same here.

Reply:
Reverted and replaced the AppendLine with Append